README.md

```diff
@@ -75,6 +75,47 @@ You can copy the [dist/accessible-autocomplete.min.js](dist/accessible-autocompl
 <script type="text/javascript" src="assets/js/accessible-autocomplete.min.js"></script>
 ```
 
+### As a GOV.UK Prototype Kit plugin
+
+In your prototype folder, run:
+
+```bash
+npm install accessible-autocomplete
+```
+
+To add the accessible autocomplete to a page in your prototype, first add a `select` component with your options. For example:
+
+```html
+<div class="govuk-form-group">
+  <label class="govuk-label" for="country">
+    Country
+  </label>
+  <select class="govuk-select" id="country" name="country">
+    <option value="England">England</option>
+    <option value="Northern Ireland">Northern Ireland</option>
+    <option value="Scotland">Scotland</option>
+    <option value="Wales">Wales</option>
+  </select>
+</div>
+```
+
```

Comment:
I think relying on the class is a bit fragile, but if thats the approach I would suggest calling it out, something like:

Note that the class `govuk-form-group` is required to make the autocomplete look like GOV.UK.

Because if they use examples from elsewhere, it won't have it.

Reply:
Maybe it could get a mention here too?

https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#styling-the-autocomplete